_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+Author: Levi Blinder
+Title: Arts and Sciences Recipe Page
+---
+
+```
+<doc>
+...
+...
+...
+<div type = "recipe">
+  ...
+  ...
+  ...
+  <page>44</page>
+  <pageNum>14</pageNum>
+  <p>
+    it down with a <tl> String </tl> as far as
+    the water, then fire the <m> Gun Powder </m>
+    with a <tl> hot Tool </tl>, + it will clean the
+    well of every noxious vapor.
+  </p>
+</div>
+
+<red> <dbl> </dbl> </red>
+
+<div type = "recipe">
+  <recNum> <red> 16 </red> </recNum>
+  <head> <red> To Kill Buggs </red> </head>
+  <line> </line>
+  <p>
+    Mix <m> Oil of Turpentine </m> with
+    the <m> <pl> Neopolitan </pl> Ointment </m>, + smear
+    the Bedstead, + peel into the <q> ottebes </q>
+    where the Buggs are + they will be
+    destroyed.
+  </p>
+</div>
+
+<red> <dbl> </dbl> </red>
+
+<div type = "recipe">
+  <recNum> <brown> 17 </brown> </recNum>
```

Comment:
I love that you've encoded the different colors here. I think that's really important for a handwritten document like a manuscript recipe book, because the changing colors reflect the fact that the document was used over time with different inks used at different times.